hadoop-hdds/interface-admin/src/main/resources/proto.lock

```diff
@@ -1334,7 +1334,7 @@
               },
               {
                 "id": 3,
-                "name": "idleiterations",
```

Comment:
Don't EVER change proto lock files! These are automatically updated by protolock only on each major or minor release to ensure compatibility. This is an easy mistake to make when proto-backwards-compatability-check fails though, and it would definitely be helpful if our CI could help the devs out here : ) @adoroszlai is there an easy check we can put in CI to fail PRs that that modify anything called proto.lock?

Reply:
Good catch. This seems to have sneaked in while I was refactoring this field's name across the project. CI got skipped since this is still a draft PR.

BTW, from what I understand, it's compatible to change the name of a proto field as long as the ID and type remain the same, and JSON is not involved. The [hadoop policy](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/Compatibility.html#Policy) that's linked [here](https://github.com/apache/ozone/blob/af4730a54bd2bbc63d556936292ae760701b7e7f/hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto#L21) also says it's compatible.

Reply:
Yeah the rename in the proto itself is safe, lets just make sure the proto lock stays in its original state.

Reply:
Changing only the proto file fails the backwards compatibility check here:

```
[INFO] CONFLICT: "StartContainerBalancerRequestProto" field: "idleiterations" has been removed, but is not reserved [ScmAdminProtocol.proto]
[INFO] CONFLICT: "StartContainerBalancerRequestProto" field: "iterations" ID: 3 has an updated name, previously "idleiterations" [ScmAdminProtocol.proto]
```

With the proto lock file in its original state.